QuizCraft.Domain.API/Program.cs

```diff
@@ -1,12 +1,30 @@
 using QuizCraft.Domain.API.APIClients;
 using QuizCraft.Domain.API.Services;
+using Microsoft.OpenApi.Models;
 
 var builder = WebApplication.CreateBuilder(args);
 
-builder.Services.AddControllers();
+builder.Services.AddCors(options =>
+{
+    options.AddPolicy("AllowAllOrigins",
+        builder =>
+        {
+            builder.AllowAnyOrigin()
```

Comment:
Maybe lets allow only `localhost:3000` for now, later we can change to a deployed URL.